spec/features/Welcome/index_spec.rb

```diff
@@ -10,4 +10,81 @@
 
     expect(current_path).to eq("/quotes")
   end
+
+  it "I see total number of quotes that were found for that keyword, search keyword, and quote info" do
+    output = {
+    "count": 1,
+    "results": [
+        {
+            "_id": "JIk3cbQ8s",
+            "author": "Michael Jordan",
+            "content": "I've missed more than 9000 shots in my career. I've lost almost 300 games. 26 times, I've been trusted to take the game winning shot and missed. I've failed over and over and over again in my life. And that is why I succeed.",
+            "tags": [
+                "sports",
+                "competition",
+                "famous-quotes"
+            ],
+            "authorSlug": "michael-jordan",
+            "length": 224,
+            "dateAdded": "2022-07-06",
+            "dateModified": "2022-07-08"
+        }]}
+
+    allow(QuotableService).to receive(:search).and_return(output)
```

Comment:
I like that you're mocking your response here rather than making a full API call for your testing. Good choice!